qa/__tests__/smoke.test.ts

```diff
@@ -137,5 +137,19 @@ describe('Smoke Tests', () => {
     expect(classicCookies['ajs_user_id']).toContain('Test')
 
     compareSchema(results)
+
+    const [url2, chrome2] = await Promise.all([server(true), browser()])
```

Comment:
Should we extract this out into its own test? feels like a separate test with new setup etc.

Reply:
It worked this time /shrug